package.json

```diff
@@ -1113,6 +1131,16 @@
                 "command": "kubernetes.portForwarding.showSessions",
                 "title": "Show Port Forwarding Sessions",
                 "category": "Kubernetes"
+            },
+            {
+                "command": "extension.ConfigureKubernetesPipeline",
+                "title": "Configure CI/CD",
+                "category": "Azure App Service"
```

Comment:
category is wrong

Reply:
what should be the category?

Reply:
Not really sure. where does this category show up to user?

Reply:
Ah this would show up in command paletter. But we are hiding this command from command palette, so it should not matter.
May be we can use "Deploy to Azure" category. That is what we do currently in our extension.